src/main/kotlin/org/malachite/estella/people/domain/HrPartnerPayloads.kt

```diff
@@ -0,0 +1,14 @@
+package org.malachite.estella.people.domain
+
+import org.malachite.estella.commons.models.people.HrPartner
+
+class HrPartnerPayloads {
```

Comment:
Dlaczego aż tak? W sensie czemu nie zostawisz po prostu data classy HrPartnerResponse?

Reply:
nie chciałem robić tak że w organizationControlerze będzie payload odnośnie hrPartnera i nie chciałem robić bałaganu w HrPartnerControlerze i definiować tam dataklasy, której nie potrzebuje

Reply:
Ale to wiesz, że możesz zrobić plik HrPartnersPayloads, dać go do katalogu `people/domain` i tam definiować data classy? W sensie nie widzę potrzeby, żebyśmy mieli nadrzędną klasę HrPartnerPayloads

Reply:
aa o to Ci chodzi, to posłusznie posłuchałem flow intelki XD

Reply:
wait what? XDD